content/zh-cn/docs/concepts/security/pod-security-standards.md

```diff
@@ -315,6 +315,7 @@ fail validation.
 					<li><code>net.ipv4.ip_unprivileged_port_start</code></li>
 					<li><code>net.ipv4.tcp_syncookies</code></li>
 					<li><code>net.ipv4.ping_group_range</code></li>
+					<li><code>net.ipv4.ip_local_reserved_ports</code>（从 Kubernetes 1.27 开始）</li>
```

Comment:
Please follow the context style to copy and comment the en text

Reply:
What's wrong with this? I think it's fine.

Reply:
Like this:

```suggestion
					<li><code>net.ipv4.ip_local_reserved_ports</code><!-- (since Kubernetes 1.27)-->（从 Kubernetes 1.27 开始）</li>
```

Reply:
done